_posts/2019/2019-12-15-use-loose-ruby-versioning-in-your-gemfile.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Use Loose Ruby Versioning in Your Gemfile"
+description: "The tiniest little improvement to your life"
+layout: article
+category: ruby
+image:
+  base: '2019/use-loose-ruby-versioning-in-your-gemfile'
+  alt: 'Red numbered dice'
+  credit:  Terry Vlisidis
+  source: "https://unsplash.com/photos/vPQbo1D7Eco"
+
+---
+
+Patch level versions (e.g. 2.6.**3**) of Ruby are compatible with each other and often only include bug fixes and security patches. Significant upgrades to the Ruby language, that include new syntax, features and make bigger changes to the language are released once a year—as a gift for us all—on Christmas Day.
+
+Many developers ensure their application use a specific version in development by using a “Ruby version manager”, like [rvm](https://rvm.io) or, my preferred choice, [chruby](https://github.com/postmodern/chruby).
+
+These version managers look for a [file](https://gist.github.com/fnichol/1912050) in the root directory of each app called `.ruby-version`. The file specifies the correct version of Ruby for this application and then switches the local environment, switching between multiple installed versions of Ruby automatically.
```

Comment:
*and then automatically switches the local environment to the required version (?)

Reply:
```suggestion
These version managers look for a [file](https://gist.github.com/fnichol/1912050) in the root directory of each app called `.ruby-version`. The file specifies the correct version of Ruby for this application and then automatically switches the local environment to the required version.
```